addons/project/models/project_task.py

```diff
@@ -944,6 +949,26 @@ def create(self, vals_list):
                 task.message_subscribe(follower.partner_id.ids, follower.subtype_ids.ids)
             if current_partner not in task.message_partner_ids:
                 task.message_subscribe(current_partner.ids)
+            if task.email_cc:
+                partners = self.env['res.partner'].search([('email', 'in', tools.email_split(task.email_cc))])
```

Comment:
We should maybe do read_group and group by email to fetch all the partners for all tasks in one query. 🙂

Reply:
I can not really do a read group, since i have to split each email_cc field of each task (in case an email cc contains more than one email). So i have to do a first loop to get the emails, then I keep the second loop and i use the results. Its better query wise, since only one query is done to fetched all the partner, but i wonder if it is not counter productive on real use case. The email cc ( as far as i've seen) is only available on form view, so the user should technically never be able to create task in batch with email_cc. On the other hand, when he creates tasks in batch, the new loop will reduce the performance compare to before. Not sure which option is best. I'll still try to apply the changes, and we'll see once its done which option you think is best.